tests/parameter_sources/elasticlogs_bulk_source_test.py

```diff
@@ -15,11 +16,6 @@ def generate_event(self):
         return self.doc, self.index, self.type
 
 
-class StaticTrack:
```

Comment:
Was this removed on purpose?

It is used in tests [here](https://github.com/elastic/rally-eventdata-track/pull/53/files#diff-77001cf4fc758355046aa067185cf830R24), [here](https://github.com/danielmitterdorfer/rally-eventdata-track/blob/321edf8aaff0a209925c73fa10b1feb1f4661c8f/tests/parameter_sources/elasticlogs_kibana_source_test.py#L31) and here: https://github.com/danielmitterdorfer/rally-eventdata-track/blob/321edf8aaff0a209925c73fa10b1feb1f4661c8f/tests/parameter_sources/elasticlogs_bulk_source_test.py#L2

Reply:
It was not removed but moved to `__init__.py`. For some reason the import worked when running in the IDE but I also noticed the tests where failing on the command line and fixed it in https://github.com/elastic/rally-eventdata-track/pull/53/commits/ccfd495cb1f78d5a6879f1b64ebb4ba086143954. Thanks for catching.